drivers/SmartThings/philips-hue/src/disco/light.lua

```diff
@@ -75,14 +75,8 @@ function M.handle_discovered_device(
         goto continue
       end
 
-      local parent_assigned_child_key
       local device_name = light.metadata.name
-      if __DBG and __DBG.emulate_dth_ids then
-        device_name = "DTH_" .. device_name
-        parent_assigned_child_key = st_utils.generate_uuid_v4()
-      else
-        parent_assigned_child_key = string.format("%s:%s", light.type, light.id)
-      end
+      local parent_assigned_child_key = string.format("%s:%s", light.type, light.id)
```

Comment:
have we always prefixed this with the `light.type`?  If we didn't, then I think we need to leave lights to not have a prefix.